packages/x-components/src/design-system/components/suggestion-group/default.scss

```diff
@@ -29,6 +28,18 @@
   gap: var(--x-size-gap-suggestion-group-default);
   @include safari-gap(var(--x-size-gap-suggestion-group-default));
 
+  // border
+  border: none;
```

Comment:
This `border:none` has the same specifity as the tag ones. This means that the border-color depends on the order the CSS is loaded. If suggestion/suggestion-group is loaded before, and the tag is loaded after, everything works. If the tag is loaded first, it fails. Is this `border: none` here for any specific reason, or can we remove it?

Reply:
I think this also happens with the `align-items: flex-start` property